cardano-node/src/Cardano/Node/Configuration/Topology.hs

```diff
@@ -47,25 +40,29 @@ data RemoteAddress = RemoteAddress
   -- ^ Either a dns address or an ip address.
   , raPort      :: !PortNumber
   -- ^ Port number of the destination.
-  , raAdvertise :: !PeerAdvertise
-  -- ^ Advertise the peer through gossip protocol.
+  , raValency :: !Int
   -- ^ If a DNS address is given valency governs
```

Comment:
Does this comment belong with the comment below? If it does, lets move the entire comment under this record.